docs/SRS-Volere/SRS.tex

```diff
@@ -77,22 +79,44 @@ \subsection{Maintenance Users and Service Technicians}
 
 \section{Mandated Constraints}
 \subsection{Solution Constraints}
-\lips
+\textbf{Mobile Device}\\
+\underline{Description}: User will interact with the system through an app on their mobile device (smartphone).\\
+\underline{Rationale}: Mobile devices are widely used and ubiquitous. While at a grocery store, users may need access to the system.\\
+\underline{Fit Criterion}: User interface will run entirely on a mobile device.\\
+\\
+\textbf{Internet access}\\ %%todo: is this too solution specific?
+\underline{Description}: User must have internet access.\\
+\underline{Rationale}: System servers and databases are used to store collective user data and process that data. User's devices are required to communicate with the servers and databases via internet access.\\
+\underline{Fit Criterion}: User interface will have internet access.\\
```

Comment:
Is this right? That the "user interface will have internet access" or is it that the user interface will display whether it has internet access? Alternatively, the system must have internet access?

Reply:
Good question. Personally I think offline activity would be a plus but if we don't want to work on it it's also ok